CHANGELOG.md

```diff
@@ -4,7 +4,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Next Version]
```

Comment:
Do you want to change ## [Next Version] to the following (or similar)?

## 24.07 Release

Reply:
That's a good reminder. I can change it to 0.4.0, which is the release in this repo

Reply:
```suggestion
## [0.4.0]
```